docs/content/doc/help/faq.en-us.md

```diff
@@ -292,15 +241,72 @@ By default, your LFS token will expire after 20 minutes. If you have a slow conn
 
 You may want to set this value to `60m` or `120m`.
 
-## How can I create users before starting Gitea
+If git reports the following when pushing, you likely didn't push through Gitea, see [SSH issues](#ssh-issues).
+```
+error: exit status 127, message: bash: git-lfs-authenticate: command not found (try: 5/6)
+```
 
-Gitea provides a sub-command `gitea migrate` to initialize the database, after which you can use the [admin CLI commands]({{< relref "doc/usage/command-line.en-us.md#admin" >}}) to add users like normal.
+### Gitea is running slow
 
-## How can I enable password reset
+The most common culprit for this is loading federated avatars.  
+This can be turned off by setting `ENABLE_FEDERATED_AVATAR` to `false` in your `app.ini`  
+Another option that may need to be changed is setting `DISABLE_GRAVATAR` to `true` in your `app.ini`
```

Comment:
So we solved this particular issue when we pushed the federated avatar lookup to when the avatar is requested.

---

The usual cause of slowdowns right now is the last commit lookup - hopefully #16467 and #16432 will fix that one - and then it will be the diff renderer which I might be able to get to if those would get reviewed and merged.